patches/sdk/0005-Don-t-build-for-net472.patch

```diff
@@ -0,0 +1,53 @@
+From 54f8bf96e5e340ae997020574f9739cb6455f121 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Sat, 20 Jun 2020 18:58:49 -0400
+Subject: [PATCH] Don't build for net472
```

Comment:
Don't we need to for VSCode? Just need those targeting packages?

Reply:
I logged https://github.com/dotnet/source-build/issues/1640 for this, since the solution is non-obvious.